lib/run_loop/environment.rb

```diff
@@ -0,0 +1,11 @@
+module RunLoop
+  class Environment
+
+    # Returns the user's Unix uid.
+    # @return [Integer] The user's Unix uid as an integer.
+    def self.uid
+      `id -u`.strip.to_i
```

Comment:
Discussed in person.

Yeah, it could, but I don't want us to chase our tail here - if `id` is broken on a user's machine, then something is seriously wrong.